ui/component/notification/index.js

```diff
@@ -1,8 +1,19 @@
 import { connect } from 'react-redux';
```

Comment:
![image](https://user-images.githubusercontent.com/64950861/127957130-5489ac4d-73ba-45cd-91b6-d79f6ea7c701.png)

The navlink only works when clicked on text.  If you middle-click on empty areas, it doesn't work.
`ClaimPreview` solves this by adding a wrapper to the blank areas. 

Not super critical -- can postpone.